apps/webapp/src/state/swap.ts

```diff
@@ -49,10 +52,18 @@ export const createSwapSlice = (): SliceCreator<SwapSlice> => (set, get) => {
       const { dismiss } = toastFn(loadingTxToast);
 
       try {
-        const req = await assembleRequest(get().swap);
-        const txHash = await planWitnessBuildBroadcast(req);
+        const swapReq = await assembleSwapRequest(get().swap);
+        const swapTx = await planWitnessBuildBroadcast(swapReq);
+        const swapCommitment = getSwapCommitment(swapTx);
+
+        const swapClaimReq = assembleSwapClaimRequest(swapCommitment);
+        const swapClaimTx = await planWitnessBuildBroadcast(swapClaimReq, {
+          rpcMethod: 'witnessAndBuild',
+        });
```

Comment:
Wonder if we should have a code comment about swap claims not requiring second auth

Reply:
Good call — I added a comment